opencog/nlp/chatbot-psi/chatbot.scm

```diff
@@ -18,51 +18,62 @@
     (cancel-all-threads)
     (reset-all-states)
 
-    (let* ((sent-node (car (nlp-parse utterance)))
-           (list-of-words (get-word-list sent-node)))
-
-        (State input-utterance
-            (Reference
-                list-of-words
-                sent-node
-                (Node utterance)
-            )
+    (catch #t
```

Comment:
I fixed #2192 just now, but I suppose that there are probably other exceptions too. They should be logged, not hidden ...